src/internal/Popover.js

```diff
@@ -281,6 +295,17 @@ class Popover extends Component<DefaultProps, Props, void> {
     }
   };
 
+  handleResize = debounce(
+    () => {
+      const element: any = ReactDOM.findDOMNode(this.transitionEl);
+      this.setPositioningStyles(element);
+    },
+    166,
+    {
+      leading: true,
```

Comment:
I added that leading option to reduce the lag in updating the position of the popover. Would you like me remove it?